src/infoview.ts

```diff
@@ -36,26 +38,34 @@ class InfoDocument implements Disposable {
     }
 
     get text() {
-        return this.lines.join('\n');
+        return this.contents;
     }
 
     dispose() {
         for (const s of this.subscriptions) s.dispose();
     }
 
     private render() {
-        this.lines = [];
-        this.renderHeader();
-        this.renderGoal();
-        this.renderMessages();
+        this.contents = this.renderHeader() + this.renderGoal() + this.renderMessages();
+    }
+
+    private stylesheet() {
+        return `#settings { display: block; text-align: right; }
+            h1 {
+                display: block;
+                background-color: blue;
```

Comment:
I'm currently playing around with having a more separate view on the goal and the list of error messages. I guess I will introduce a split view, with a fixed (user setable) size for the goal view.

@jroesch I mostly try to stick to this, but I'm not sure how we can reuse existing theme colors. For example I want to have different colors for the message titles, depending if they are errors, warnings or informations.